pkg/accounting/tally/tally.go

```diff
@@ -65,6 +68,15 @@ func (t *Service) Run(ctx context.Context) (err error) {
 
 // Tally calculates data-at-rest usage once
 func (t *Service) Tally(ctx context.Context) error {
+	// The RT cache will only keep deltas to space/bw used relative to the
```

Comment:
its hard to know what "RT" stands for. If possible, could we say "real time" in places where "RT" is?

Reply:
I am very amenable to that, but first can somebody come up with a better name than `accountingRTCache` for that member? I've had an especially hard time naming this whole subsystem because it doesn't really fill any normal role that people would be familiar with. It's not _really_ a cache, and it can't be too similar to `accountingDB` because that would cause confusion with a different thing that is related but entirely different. `accountingRealTimeCache` would imply that it's a "real time cache", whatever that is, instead of a collection of live accounting data used for real-time tracking purposes.

Reply:
What about `liveaccounting`?

Reply:
Works for me.